test/integration.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
```

Comment:
@duglin That has Docker container volume mounting that would mess with "Docker-outside-of-Docker". I think we should be able to build/test without relying on Docker, but having it as the default for simplicity.